pkg/rancher-desktop/config/settings.ts

```diff
@@ -391,9 +394,39 @@ const updateTable: Record<number, (settings: any) => void> = {
     // The updater still wants to see an entry here (for updating ancient systems),
     // but will no longer delete obsolete files.
   },
-  3: (settings) => {
-    // Should stay true until the kim-based buildkit artifacts are removed -- see code in lima.ts:start()
-    settings.kubernetes.checkForExistingKimBuilder = true;
+  3: (_) => {
+    // With settings v5, all traces of the kim builder are gone now, so remove this line:
+    // settings.kubernetes.checkForExistingKimBuilder = true;
```

Comment:
Please don't leave commented code around.

Note that this means that we should either keep this as-is, or just delete the case (with a comment about why we skip 3).

Reply:
Leaving the explanatory comment in place, although it could have been like the change leaving version 1.